### PR TITLE
[DOC]PR for Missing information for xcluster ports 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6083

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@shahrooz1997 Could you, please, take a quick look at this doc PR ?

File: docs/content/preview/reference/configuration/default-ports.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Internode (server-to-server or node-to-node) communication is managed using RPC
 
 To enable login to the machines running these servers, the SSH port 22 should be opened.
 
+xCluster usesthe YB-Master port 7100 for the initial communication and then uses the YB-TServer port 9100 to get data changes.
```
Comment (shahrooz1997) [inline docs/content/preview/reference/configuration/default-ports.md:37]: `usesthe` should be `uses the`

**Review by shahrooz1997 — APPROVED** [review]
It needs a minor change.

File: docs/content/preview/reference/configuration/default-ports.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/reference/configuration/default-ports.md]: fixed

**Pull request merged**